packages/app/src/elements/menu.ts

```diff
@@ -558,6 +558,16 @@ export class Menu extends Routing(StateMixin(LitElement)) {
                         <div class="stretch">${$l("Password Generator")}</div>
                     </div>
 
+                    <div
+                        class="menu-item"
+                        @click=${() => this._goTo("audit")}
+                        aria-selected=${this.selected === "audit"}
+                    >
+                        <pl-icon icon="shield-check"></pl-icon>
+
+                        <div class="stretch">${$l("Security Audit")}</div>
```

Comment:
I like this name, but wouldn't oppose `Security Check` (I've seen your comment). If you want me to change it, let me know if I should change the file names/filter, etc.

Reply:
Another idea: `Security Scanner`?

Reply:
I still prefer `Security Audit` or `Security Check`.

Reply:
Ok, let's stick with `Security Audit` then 😊